docs/source/Email.md

```diff
@@ -0,0 +1,68 @@
+# Email Sending
```

Comment:
Let's have this section renamed to simply "Email". What do you think?

Reply:
Ok, "Email" it is.